eventonica-api/routes/events.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+const cors = require('cors');
+const router = express.Router();
+const Event = require('../models/event');
+const { Op } = require("sequelize");
+
+router.use(express.json());
+router.use(cors());
+router.use(express.urlencoded({ extended: true }))
+
+//GET list of events
+router.get('/', (req, res) => {
+  Event.findAll()
+  .then(events => res.json(events))
+  .catch(error => {
+    console.log('ERROR:', error)
+  });
+});
+
+//ADD new event
+router.post('/add', (req, res) => {
+  const newEvent = req.body;
+  Event.create({ 
+    id: newEvent.id, 
+    name: newEvent.name, 
+    date: newEvent.date,
+    description: newEvent.description,
+    category: newEvent.category
+   });
+  res.json(events);
+});
+
+//DELETE an existing event
+router.delete('/:deleteEventId', (req, res) => {
+  const deleteEventId = req.params.deleteEventId;
+  Event.destroy({
```

Comment:
It would be nice to handle the case where you pass an invalid id and return a 404.